.github/workflows/go.yml

```diff
@@ -865,6 +865,43 @@ jobs:
           path: |
             /tmp/*.log
 
+  smoketest-extraargs:
```

Comment:
I start thinking that might be we need to have codegen tool for adding new e2e test suites 🙂

Reply:
I'm thinking if we could have something that "triggers" each smoke as a separate workflow or something. So basically after the build artifact is store in cache a separate step could then kick off each smoke as a separate workflow.